ckanext/recombinant/views.py

```diff
@@ -571,6 +601,48 @@ def preview_table(resource_name: str,
         })
 
 
+@recombinant.route('/recombinant/refresh/<resource_name>/<owner_org>',
+                   methods=['GET', 'POST'])
+def refresh_dataset(resource_name: str, owner_org: str):
+    if request.method != 'POST':
+        # handle page refreshes
+        h.flash_notice(_('Form not submitted, please try again.'))
+        return h.redirect_to('recombinant.preview_table',
+                             resource_name=resource_name,
+                             owner_org=owner_org)
+    if 'confirm' in request.form:
+        dataset_id = request.form['dataset_id']
+        if not dataset_id:
+            h.flash_error(_('Could not determine dataset to update.'))
+        else:
+            lc = LocalCKAN()
+            dataset_dict = lc.action.package_show(id=dataset_id)
+            for res in dataset_dict['resources']:
+                if not h.check_access('datastore_delete',
+                                      {'resource_id': res['id'],
+                                       'filters': {}}):
+                    return abort(403, _('User {0} not authorized '
+                                        'to update resource {1}'.format(
+                                            str(g.user), res['id'])))
+            try:
+                lc.action.recombinant_update(
+                    owner_org=dataset_dict['organization']['name'],
+                    dataset_type=dataset_dict['type'],
+                    dataset_id=dataset_id,
+                    force_update=True)
```

Comment:
Hmm. Are there situations where we wouldn't want a user to run recombinant_update with force=True?  Maybe while in the middle of a data migration on our end?

Reply:
Yes that sounds like a safer default, especially if it can result in data loss.